lib/board.rb

```diff
@@ -148,8 +148,24 @@ def place(ship, coordinates)
   end
 
   def render(value = false)
-
-    #@cells.each do |cell|
+    print "  1 2 3 4 \n"
```

Comment:
Potential note for refactoring - we can make this into a loop as well and print from @columns when we want to make the board size variable